client/components/dashboard/dashboard-service.js

```diff
@@ -416,17 +417,17 @@ DashboardService.prototype.selectWidget = function(
     }
   }
 
-  if (widget) {
+  if (goog.isDefAndNotNull(widget)) {
     this.sidebarTabService_.resolveSelectedTabForWidget();
 
     this.timeout_(() => {
-      this.scrollWidgetIntoView(widget);
+      this.scrollWidgetIntoView(/** @type {!WidgetConfig} */ (widget));
```

Comment:
Are you sure you need the cast? I thought Closure Compiler does some type inferencing and should be able to deduce it's not null.